.vib/wordpress/cypress/cypress/integration/wordpress_spec.js

```diff
@@ -2,88 +2,35 @@
 
 import { random } from './utils';
 
-it('contains the sample blogpost and a comment', () => {
+it('allows login/logout', () => {
+  cy.login();
+  cy.get('#login_error').should('not.exist');
+  cy.visit('wp-login.php?loggedout=true&wp_lang=en_US');
+  cy.contains('.message', 'logged out');
+});
+
+it('allows adding a comment to sample blogpost', () => {
   cy.visit('/');
   cy.get('.wp-block-query');
   cy.get('.wp-block-post-title a').click();
-  cy.fixture('helloworld').then((hw) => {
-    cy.contains('.wp-block-post-title', hw.title);
-    cy.contains('.wp-block-post-content', hw.content);
-  });
-
-  cy.get('.wp-block-post-title').click();
-  cy.get('.commentlist').should('have.length', 1);
-  cy.fixture('helloworld').then((hw) => {
-    cy.get('.comment-author').should('contain.text', hw.commenter);
-
-    cy.fixture('helloworld').then((hw) => {
-      cy.get('#comment').type(hw.comment).should('have.value', hw.comment);
-    });
-
-    cy.fixture('user').then((user) => {
-      cy.get('#author').type(user.username).should('have.value', user.username);
-      cy.get('#email').type(user.email).should('have.value', user.email);
-    });
-
-    cy.get('#submit').click();
+  cy.fixture('posts').then((post) => {
+    cy.contains('.wp-block-post-title', post.title);
+    cy.contains('.wp-block-post-content', post.content);
+    cy.get('.wp-block-post-title').click();
+    cy.get('.commentlist').should('have.length', 1);
```

Comment:
Would this work if the test is executed multiple times?

Reply:
Yeah, it does! However I want to rethink this test, you have a good point.
It seems to be working because the comment is always added to a new post.
Since we are creating posts all the time, there is always one comment per post if that makes sense.

Reply:
I'm not sure why we need to check the comments list here. If we are not checking whether its length increases after adding a comment at the end of the test, there is not really a reason to do so at the start right?

Reply:
We are adding a comment each time. 
We want to check if the comment is added properly. 
The length **doesn't** increase because every comment added **needs** to be approved by the moderator. So each time the comment is added it will appear to be the first one.

Reply:
I think I made a mistake in the previous explanation. Let me try to explain it better
In this test, we first check for the presence of the Hello World comment in the Hello World post. Then we proceed to add our own comment which then goes to approval.
This test verifies if the Hello World comment is there before proceeding to add its own. I think it's a good way to check if the comment section is loaded properly, as well as a good way to check if the comment section works properly. UI for reference:

![image](https://user-images.githubusercontent.com/47592680/167843325-05b6e81d-e00f-40d7-bfd9-d15e3f8752d3.png)

P.S. This test can be executed as many times as possible without issues, which was the original question.

Reply:
nice, now I got you! thanks 👍